addons/web/static/src/js/fields/abstract_field.js

```diff
@@ -408,6 +408,7 @@ var AbstractField = Widget.extend({
             changes: changes,
```

Comment:
update doc for option

Reply:
Wouldn't it be cognitively easier to have the option be notifyChange=true?